transcom/mymove
# MB-16476: Add deprecated and error statuses to TIO queue

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16476)

## Summary

This adds the statuses of deprecated and error to list of checkboxes from which you can filter payment requests.

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the office app 
2. Login as a TIO user
3. If you click the status header filter section you should see the addition of the deprecated and error statuses.
